test/functional/Java8andUp/playlist.xml

```diff
@@ -1920,4 +1920,23 @@
 		</subsets>
 	</test>
 
+	<test>
+		<testCaseName>loadLegacyLibrary</testCaseName>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) $(JAVA_COMMAND) $(JVM_OPTIONS)  \
+			-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+			org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+			-testnames loadLegacyLibrary \
+			-groups $(TEST_GROUP) \
+			-excludegroups $(DEFAULT_EXCLUDE); \
+			$(TEST_STATUS)
+		</command>
+		<platformRequirements>os.osx</platformRequirements>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
```

Comment:
- Since `loadLibraryTest` lib is needed for the test, please add the following:
```
<types>
	<type>native</type>
</types>
```

- Is this an openj9 specific test? If so, please add the following:
```
<impls>
	<impl>openj9</impl>
</impls>
```

Reply:
added `native` tag